web/src/main/java/edu/softserve/zoo/dto/LoginRequestDto.java

```diff
@@ -0,0 +1,39 @@
+package edu.softserve.zoo.dto;
+
+import edu.softserve.zoo.annotation.IrrespectiveDto;
+
+/**
+ * Represents information passed by a user during login attempt.
+ *
+ * @author Ilya Doroshenko
+ */
+@IrrespectiveDto
+public class LoginRequestDto {
```

Comment:
i think i get the overall concept of it and what purposes that protocol was designed for